### PR TITLE
Blog get, update and delete

### PR DESCRIPTION
**Pull request by MAHMUDSWE**

### CODE REVIEW

File: backend/controllers/auth.controller.js [hunk 1]
```diff
@@ -25,10 +29,13 @@ const userLogin = async (req, res, next) => {
 
         const token = await authService.userLogin(loginCredentials);
 
+        const access_token = token;
+
         authUtils.setTokenToHeader(token, res);
```
Comment (RakibulRanak) [inline backend/controllers/auth.controller.js:34]: call setTokenHeader in authService

File: backend/respositories/user.repository.js [hunk 2]
```diff
@@ -60,30 +36,18 @@ const getUserByEmail = async (email) => {
...
-    return new UserDTO(user);
+    return user;
 }
 
 const updateUser = async (user, updateFields) => {
 
-    var result = await User.update(updateFields, {
-        where: { userid: user.userid }
-    });
+    var updatedUser = await user.update(updateFields);
```
Comment (RakibulRanak) [inline backend/respositories/user.repository.js:44]: always declare variables with const, if you realize that the value of the variable needs to change, go back and change it to let.

**Review by RakibulRanak — CHANGES_REQUESTED** [review]

File: backend/controllers/auth.controller.js (hunk 1 above)
Comment (MAHMUDSWE) [reply backend/controllers/auth.controller.js:34]: solved

File: backend/respositories/user.repository.js (hunk 2 above)
Comment (MAHMUDSWE) [reply backend/respositories/user.repository.js:43]: changed

**Review by RakibulRanak — APPROVED** [review]

**Pull request merged**